### PR TITLE
app skeleton built, core functionality of page works, some other updates as well

### PR DESCRIPTION
**Pull request by samlawson355**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -5,7 +5,19 @@ const port = 7777;
 const db = require("../database/index.js");
 
 app.use(express.static(path.join(__dirname, "../public/dist")));
-
 app.use(express.json());
 
+app.get("/5", (req, res) => {
```
Comment (timsand) [inline server/index.js:10]: Is this route (/5) for testing?

**Review by timsand — APPROVED** [review]
Looks good. Do you have to clean the description every time (not sure what your API calls look like)?

File: server/index.js (hunk 1 above)
Comment (samlawson355) [reply server/index.js:10]: Yeah, that's for testing. I hard-coded a few things before knowing how the proxy routing will look

**Comment by samlawson355** [comment]
> i

I do, I wasn't able to store bullet-pointed lines in mysql, so I have to manually break up the lines by using ++ as a line-break character. I'm sure there is a better way to do this, so if you know of one, I am all ears! Thanks man


> Looks good. Do you have to clean the description every time (not sure what your API calls look like)?

**Pull request merged**